### PR TITLE
Create the website for mobile layout

### LINKED ISSUE #142: Mobile layout
The site is not mobile-friendly at all. Any type of improvement is welcome

### PR DESCRIPTION
**Pull request by prashantug21**
# Pull Request Type
[] New feature
[] Bug fix

**This PR closes #142 **

# 👨‍💻 Describe your changes
I have update some code in the repo to make the website  responsive

# ✔️ Check List (Check all the applicable boxes)

mobile-layout-ss

- [x] I tested my code
- [x] The title and description of the PR is clear and explains what has been done.

# Additional notes

# 📷 Screenshots

### CODE REVIEW

File: .github/workflows/auto_assign.yml
Comment (mslepko) [inline .github/workflows/auto_assign.yml:1]: Please revert this change as this is not a part of the issue. 

MY_PERSONAL_ACCESS_TOKEN just expired that is why the GH action is failing, I'll handle it separately

**Review by mslepko — CHANGES_REQUESTED** [review]
It's looking great, please just revert workflow changes and I'll merge it

**Comment by prashantug21** [comment]
Done

File: .github/workflows/auto_assign.yml
Comment (prashantug21) [reply .github/workflows/auto_assign.yml:1]: I have reverted it

**Review by mslepko — APPROVED** [review]

**Pull request merged**